CONTRIBUTING.md

```diff
@@ -49,10 +49,22 @@ this will get rid of the errors.
 
     The errors should now be gone.
 
+### C) Setting up the JDT Language Server in JDT Language Server
+
+1) Please, download [Eclipse PDE support][1], open the `.vsix` file with a ZIP programm and extract the folder `/extension/server/` into a local directory.
+
+2) Ensure that [the initialize request][2] contains all paths to the extracted files in the `bundles` parameter.
+
+	If you are using [`nvim-jdtls`](https://github.com/mfussenegger/nvim-jdtls) you might have a look at [`nvim-jdtls-bundles`](https://gitlab.com/schrieveslaach/nvim-jdtls-bundles) that automates the process.
+
```

Comment:
Once you follow these instructions, JDTLS just works with PDE support in nvim? That sounds great!

Reply:
Yes, works perfectly fine.